emerge-gitclone

```diff
@@ -43,7 +43,8 @@ for repo in portage.db[eroot]['vartree'].settings.repositories:
 	# commit with the name, because the branch was not checked out.
 	# In that case we need to make the commit point to the exact
 	# remote branch name, like "refs/remotes/origin/flatcar-build-x".
-	commit = commit.replace('refs/heads/', 'refs/remotes/origin/')
+	if commit:
+		commit = commit.replace('refs/heads/', 'refs/remotes/origin/')
```

Comment:
Can we add a else case here were we print a warning?
```
else:
    print("Warning: No revision found for " + repo.sync_uri)
```

Reply:
Done

Reply:
Thanks